cmdstanpy/stanfit.py

```diff
@@ -124,6 +126,37 @@ def stepsize(self) -> np.ndarray:
             self._assemble_sample()
         return self._stepsize
 
+    @property
+    def is_optimizing(self) -> bool:
+        """Returns true if we are optimizing rather than sampling"""
+        return self._is_optimizing
+
+    @property
```

Comment:
is there a reason why first draw would be invalid for any of the other methods?
(I've seen R users hack together exactly this function - e.g. as part of simulation-based model development workflow).

Reply:
I've kept all `optimized_params_*` methods working in sampling context, yet I wonder if the naming is fine